src/Altinn.Platform/Altinn.Platform.Events/Events/Startup.cs

```diff
@@ -114,6 +118,27 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
         {
             _logger.LogInformation("Startup // Configure");
 
+            ConsoleTraceService traceService = new ConsoleTraceService { IsDebugEnabled = true };
+
+            string workspacePath = Path.Combine(Environment.CurrentDirectory, "Migration");
+
+            if (Environment.CurrentDirectory.Contains("Test"))
+            {
+                workspacePath = Path.Combine(Environment.CurrentDirectory, @"..\..\..\Migration");
+            }     
```

Comment:
The workspace path could be a configuration value. That way it can be changed for tests without this test dependent code.